dapps/marketplace/src/components/growth/Action.js

```diff
@@ -134,7 +134,7 @@ function Action(props) {
     title = fbt('Join us on Telegram', 'RewardActions.followOnTelegram')
     allowInteractionWhenCompleted = true
     // TODO: Move screen name to Enviroment variable
-    externalLink = 'https://web.telegram.org/#/im?p=@originprotocol'
+    externalLink = 'tg://resolve?domain=@originprotocol'
```

Comment:
Is this only used on mobile since I assume it won't work on mobile ?

Reply:
It does work on Desktop. But you have to have Telegram Desktop app installed. Perhaps we can have a fallback method that opens external window/tab in desktop